### PR TITLE
Upgrade wine9.0 and add patches

### PR DESCRIPTION
**Pull request by lunarlettuce**

### CODE REVIEW

**Comment by lunarlettuce** [comment]
bot, build org.vinegarhq.Vinegar

**Comment by PhoenixCausesOof** [comment]
The build fails at the “Validate build” step.

```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions repo repo
 in dir /srv/buildbot/worker/build-x86_64-4/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions repo repo'
 using PTY: False
{
    "errors": [
        "desktop-file-is-hidden"
    ],
    "warnings": [
        "appstream-summary-too-long",
        "appstream-screenshot-missing-caption"
    ]
}
program finished with exit code 1
elapsedTime=56.919080
```

So, apparently the linter detects that the desktop file is hidden? Not sure how that happened — the patches are completely unrelated to this.

Oh well, I've never worked with Flatpak before, so I don't have a say.

**Comment by lunarlettuce** [comment]
bot, build org.vinegarhq.Vinegar

File: org.vinegarhq.Vinegar.metainfo.xml [hunk 1]
```diff
@@ -23,7 +23,19 @@
 	  </screenshot>
 	</screenshots>
 	<releases>
-		<release version="1.6.0" date="2023-12-21">
+		<release version="1.6.1" date="2024-1-18">
+				<description>
+				<ul>
+					<li>Fixed default channel check for manifest path</li>
+					<li>Added matrix badge</li>
+					<li>Makefile now removes svg on uninstallation</li>
```
Comment (apprehensions) [inline org.vinegarhq.Vinegar.metainfo.xml:31]:
```suggestion
```

File: org.vinegarhq.Vinegar.metainfo.xml [hunk 2]
```diff
@@ -23,7 +23,19 @@
...
+					<li>Fixed package downloading</li>
+					<li>AVX block is now a warning</li>
+					<li>Changed sysinfo output</li>
+					<li>Unhid the .desktop for compatibility with flathub rules</li>
```
Comment (apprehensions) [inline org.vinegarhq.Vinegar.metainfo.xml:35]:
```suggestion
```

**Review by apprehensions — CHANGES_REQUESTED** [review]

**Comment by nyabinary** [comment]
NixOS depends on this PR for their package  :3

**Pull request merged**